docs/integrations/README.md

```diff
@@ -42,7 +42,7 @@ Airbyte uses a grading system for connectors to help users understand what to ex
 |[Google Analytics](./sources/googleanalytics.md)| Beta |
 |[Google Analytics v4](./sources/google-analytics-v4.md)| Beta |
 |[Google Directory](./sources/google-directory.md)| Certified |
-|[Google Search Console](./sources/google-search-console.md)| Beta |
+|[Google Search Console](./sources/google-search-console.md)| Alpha |
```

Comment:
should still be beta since we are committed to supporting it


```suggestion
|[Google Search Console](./sources/google-search-console.md)| Beta |
```